README.md

```diff
@@ -461,22 +527,7 @@ $ provenanced query wasm contract-state smart "$CONTRACT_ADDRESS" \
 ```bash
 $ provenanced query wasm contract-state smart "$CONTRACT_ADDRESS" \
   '{"get_version_info":{}}' \
-  --testnet
-```
-
-### Ask order information
```

Comment:
why remove this?

Reply:
move to https://github.com/FigureTechnologies/ats-smart-contract/pull/115/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R465 and https://github.com/FigureTechnologies/ats-smart-contract/pull/115/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R439

since when get here order already matched ...